plugins/UM3NetworkPrinting/src/ClusterUM3OutputDevice.py

```diff
@@ -140,6 +140,11 @@ def _spawnPrinterSelectionDialog(self):
         if self._printer_selection_dialog is not None:
             self._printer_selection_dialog.show()
 
+    ##  Whether the printer that this output device represents supports print job actions via the local network.
+    @pyqtProperty(bool, constant=True)
+    def supportsPrintJobActions(self) -> bool:
```

Comment:
Shouldn't this be moved to a parent class and set to true in case of this specific one?

Reply:
It's only needed in the network plugin so far, and within that plugin the output devices do not share a common class. If Cura team things this should be added to some base output device class in Cura, go ahead.

Reply:
Oh, I assumed that they did share a common ancestor. Nevermind then.